application.yml

```diff
@@ -4,7 +4,6 @@
 server.port: 0
 
 # embedded database init, supports mysql too trough the 'mysql' spring profile
-petclinic.database: hsqldb
```

Comment:
Nice catch. You confirm my PR https://github.com/spring-petclinic/spring-petclinic-microservices-config/pull/7